pkg/resources/kafka/poddisruptionbudget.go

```diff
@@ -73,8 +78,16 @@ func (r *Reconciler) computeMinAvailable(log logr.Logger) (intstr.IntOrString, e
 		Max(1, brokers-brokers*percentage) - for a percentage budget
 
 	*/
+
+	podList := &corev1.PodList{}
+	matchingLabels := client.MatchingLabels(kafka.LabelsForKafka(r.KafkaCluster.Name))
```

Comment:
I think with this label filter the cruise control pod also will be selected.

Reply:
Hi @bartam1 and thank you for your review!
If we would consider only healthy / ready pods, we might end up allowing multiple pods removal. Here is the use-case:
1. We start a Kafka rolling restart (for a cluster with a disruption = 1)
2. 1 Pod is not starting (stuck in Pending or Init due to some issues)
3. The pod from (2) is not considered by the PDB update
4. We allow a disruption of another Pod. BUt with the one from (2) we now how 2 pods down.

It's the same issue as @stoader mentioned. However, if Sebastian's case is less likely to happen, this case described here is very likely. And that's the reason for which I considered (in my previous implementation) all the Pods, regardless of their state.